src/test/java/com/exasol/adapter/dialects/postgresql/installer/InstallerIT.java

```diff
@@ -0,0 +1,108 @@
+package com.exasol.adapter.dialects.postgresql.installer;
+
+import static com.exasol.matcher.ResultSetStructureMatcher.table;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.concurrent.TimeoutException;
+
+import org.apache.commons.cli.ParseException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.containers.ExasolContainer;
+import com.exasol.matcher.TypeMatchMode;
+
+@Tag("integration")
+@Testcontainers
+class InstallerIT {
+    private static final String EXASOL_SCHEMA_NAME = "ADAPTER";
+    private static final String EXASOL_ADAPTER_NAME = "POSTGRES_ADAPTER_SCRIPT";
+    private static final String POSTGRES_JDBC_CONNECTION = "POSTGRES_JDBC_CONNECTION";
+    public static final String POSTGRES_SCHEMA = "postgres_schema";
+    public static final String SIMPLE_POSTGRES_TABLE = "simple_postgres_table";
+    public static final String EXASOL_DOCKER_IMAGE_REFERENCE = "7.0.5";
+    public static final String POSTGRES_CONTAINER_NAME = "postgres:13.1";
+
+    @Container
+    private static final PostgreSQLContainer<? extends PostgreSQLContainer<?>> POSTGRES = new PostgreSQLContainer<>(
+            POSTGRES_CONTAINER_NAME);
+    @Container
+    private static final ExasolContainer<? extends ExasolContainer<?>> EXASOL = new ExasolContainer<>(
+            EXASOL_DOCKER_IMAGE_REFERENCE).withReuse(true);
+
+    @BeforeAll
+    static void beforeAll() throws SQLException {
+        final Statement statementPostgres = POSTGRES.createConnection("").createStatement();
+        statementPostgres.execute("CREATE SCHEMA " + POSTGRES_SCHEMA);
+        createPostgresTestTableSimple(statementPostgres);
+    }
+
+    private static void createPostgresTestTableSimple(final Statement statementPostgres) throws SQLException {
+        final String qualifiedTableName = POSTGRES_SCHEMA + "." + SIMPLE_POSTGRES_TABLE;
+        statementPostgres.execute("CREATE TABLE " + qualifiedTableName + " (x INT)");
+        statementPostgres.execute("INSERT INTO " + qualifiedTableName + " VALUES (1)");
+    }
+
+    @Test
+    void testInstallVirtualSchema()
+            throws SQLException, BucketAccessException, InterruptedException, TimeoutException, ParseException {
+        final String virtualSchemaName = "POSTGRES_VIRTUAL_SCHEMA_1";
+        final String[] args = new String[] { //
+                "-virtualSchemaJarName", "virtual-schema-dist-9.0.1-postgresql-2.0.0.jar", //
```

Comment:
This library we use supports both... You can use `-virtualSchemaJarName` or `--virtualSchemaJarName` or `-vsjn` (if it would be the short tag) or `--vsjn`.  Even if I replace it In the test, the library still supports it, I don't control it

Reply:
I've just tested it, and looks like --virtualSchemaJarName is not allowed now, because I only specified short tags without 'long tags'. But I don't want to provide the second option right now. But in release droid when I provided short and long tags, it worked with both - and --